accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/installhook/AcToolInstallHook.java

```diff
@@ -69,17 +69,10 @@ public void execute(InstallContext context) throws PackageException {
                 }
 
                 if (!history.isSuccess()) {
-                    for (HistoryEntry entry : history.getErrors()) {
-                        log(entry.toString(), context.getOptions());
-                    }
-                    throw new PackageException(
-                            "Could not install configurations. Check log for detailed error message!");
+                    throw new PackageException("AC Tool installation failed with "
+                            +history.getErrors().size()+ " errors. Check log for detailed error message(s)!");
                 } else {
-                    // convert to correct (HTML) linebreaks for the package manager
-                    String log = history.toString().replaceAll("\\\n", "<br />");
-                    log(log, context.getOptions());
```

Comment:
Removed "aggregated one line log" - this was still in in addition to now proper logging